.pre-commit-config.yaml

```diff
@@ -29,24 +29,16 @@ repos:
         language: script
       - id: goimports
         name: goimports
-        entry: goimports
-        files: .go$
-        args:
-          - -l
-          - -w
-        language: system
+        entry: golangci-lint run --enable-only goimports --fix
+        types: [go]
+        language: golang
         pass_filenames: true
       - id: lint
-        name: revive go lint
-        entry: revive
-        args:
-          - "-config"
-          - "hack/revive.toml"
-          - "-exclude"
-          - "src/cmd/viper.go"
-        files: .go$
-        language: system
-        pass_filenames: true
+        name: golangci-lint go lint
+        entry: golangci-lint run
```

Comment:
Ready to approve this PR, but thoughts on using the `--new` flag here to speed up lint time? I know it they say here that linters like unused won't work with `--new` but I wasn't able to reproduce that https://github.com/golangci/golangci-lint/blob/master/.pre-commit-hooks.yaml#L3

Reply:
I would rather have all linter rules run, as that is the way it is going to run in the CI. You are not going to save any time if unused would have failed if run, and you will have to wait for the CI to run before you figure that out.

Reply:
@AustinAbro321 for your own local dev, you can always edit your `.pre-commit-config.yaml` to add the `--new` and just not commit it.  Not the best solution, but it should satisfy all parties for now, thoughts?

Reply:
Yeah I'm fine with it running without the lint flag just wanted to get Phillips thoughts. Approving now.